app/views/messages/_messages_for_library.html.erb

```diff
@@ -1,24 +1,48 @@
 <div class="messages">
-<% messages = @messages.select { |m| m.library == library_code && m.request_type == request_type } %>
-<% messages.each do |message| %>
+
+  <% messages = @messages.select { |m| m.library == library_code && m.request_type == request_type } %>
+
+  <% if library_code == 'SAL' %>
+    <% library_code = "SAL1&2" %>
```

Comment:
Well, the spec looks for SAL1&2 but since I am dynamically generating the h2's out of the library_code and request_type it doesn't quite match what's in the spec (and presumably what we want the user to see).